keel-titus-plugin/src/test/kotlin/com/netflix/spinnaker/keel/titus/resource/TestUtils.kt

```diff
@@ -36,11 +39,11 @@ fun TitusServerGroup.toClouddriverResponse(
       cloudProvider = CLOUD_PROVIDER,
       moniker = parseMoniker("$name-v$sequence"),
       env = env,
-      constraints = constraints,
-      migrationPolicy = migrationPolicy,
+      constraints = constraints.run { Constraints(hard, soft) },
```

Comment:
Would it be useful/better to add converter methods to the the source classes, perhaps (`contraints.toKeelModel()` or something?).